.github/workflows/nightly-build-example-tests-ppml.yaml

```diff
@@ -0,0 +1,102 @@
+name: Nightly Build Example Tests PPML Spark Local on Graphene
+
+on:
+  schedule:
+    - cron: '0 17 * * *'
+  workflow_dispatch:
+    inputs:
+      select_running_example:
+        description: 'select which example to run'
+        required: true
+        default: 'all'
+        type: choice
+        options:
+        - all
+        - spark_arrow
+      sgx_mem_size:
+        description: 'memeory size limit'
+        required: true
+        default: 32G
+        type: choice
+        options:
+        - 16G
+        - 32G
+        - 64G
+        - 128G
+      image:
+        description: 'docker image version'
+        required: true
+        default: 10.239.45.10/arda/intelanalytics/bigdl-ppml-trusted-big-data-ml-python-graphene:latest
+        type: string
+
+  
+jobs:
+  example-tests-ppml:
+    runs-on: [self-hosted, SGX, Wilwarin]
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: set variable
+      env:
+        DEFAULT_EXAMPLE: 'ALL'
+        DEFAULT_SGX_MEM_SIZE: 32G
+        DEFAULT_IMAGE: 10.239.45.10/arda/intelanalytics/bigdl-ppml-trusted-big-data-ml-python-graphene:latest
+      run: |
+        echo "SELECT_RUNNING_EXAMPLE=${{ github.event.inputs.select_running_example || env.DEFAULT_EXAMPLE }} " >> $GITHUB_ENV
+        echo "SGX_MEM_SIZE=${{ github.event.inputs.sgx_mem_size || env.DEFAULT_MEM_SIZE }}" >> $GITHUB_ENV
+        echo "IMAGE=${{ github.event.inputs.image || env.DEFAULT_IMAGE }}" >>$GITHUB_ENV
+    - name: start container
+      run: |
+        set -x
+        docker pull ${IMAGE}
+        docker rm -f ${CONTAINER_NAME}
+        docker run -id --privileged --net=host --name ${CONTAINER_NAME} \
+        --cpuset-cpus=$CPUSET \
+        --oom-kill-disable \
+        --device=/dev/sgx/enclave \
+        --device=/dev/sgx/provision \
+        -v ~/glorysdj/kuberconfig:/root/.kube/config \
+        -v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+        -v $ENCLAVE_KEY:/graphene/Pal/src/host/Linux-SGX/signer/enclave-key.pem \
+        -v $DATA_PATH:/ppml/trusted-big-data-ml/work/data \
+        -v $KEYS_PATH:/ppml/trusted-big-data-ml/work/keys \
+        -e LOCAL_IP=$LOCAL_IP \
+        -e SGX_MEM_SIZE=$SGX_MEM_SIZE \
+        -e NotebookPort=12345 \
+        -e NotebookToken="1234qwer" \
+        -e RUNTIME_SPARK_MASTER=k8s://https://192.168.0.112:6443 \
```

Comment:
k8s master can be set as input with default value k8s://https://192.168.0.112:6443